packages/strapi-plugin-i18n/admin/src/containers/Initializer.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ * Initializer
+ *
+ */
+
+import { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import pluginId from '../pluginId';
+import useLocales from '../hooks/useLocales';
+
+const Initializer = ({ updatePlugin }) => {
+  const { isLoading, locales } = useLocales();
+  const ref = useRef();
+
+  ref.current = updatePlugin;
+
+  useEffect(() => {
+    if (!isLoading && locales.length > 0) {
```

Comment:
Are we sure we will always have at least one locale in a project?

I think it's safer to make the following condition: `!isLoading`, WDTY?

Reply:
Yep you're right, but isLoading won't be enough, since on the first rendering, it will be false. I will add an extra steps to this (putting the locales list to undefined for instance)

Reply:
In this initial state isLoading should be true.